### ISSUE TITLE
Document lack of aria-expanded support?

### ISSUE THREAD

**Issue by lukewarlow** [keithamus/invokers-polyfill#51]
This polyfill doesn't set aria-expanded when the built-in would be. This should be fixed or documented.
